### PR TITLE
vochain: remove commitment and reveal keys

### PR DESCRIPTION
**Pull request by p4u**
The current zkSnarks circuit implemented does not use
reveal and commitment keys, so this commit removes them
from the Process data structure and from the keykeeper.

Signed-off-by: p4u <pau@dabax.net>

### CODE REVIEW

**Comment by jordipainan** [comment]
Just few points :)

- On `types/consts.go` L134 change comment to -> `// KeyKeeperMaxKeyIndex is the maxim number of allowed Encryption keys`
- On `vochain/keykeeper/keykeeper.go` OnProcessStatusChange(...) change comment to -> `// OnProcessStatusChange will publish the private keys of the ended process, if required`
- On `vochain/keykeeper/keykeeper.go` generateKeys(...) change comment to -> `// Generate Keys generates a set of encryption keys for a process. Encryption private key = hash(signer.privKey + processId + keyIndex).`
- On `vochain/keykeeper/keykeeper.go` revealKeys(...) remove commented code.

Thank you !

File: vochain/keykeeper/keykeeper.go [hunk 1]
```diff
@@ -210,7 +196,7 @@ func (k *KeyKeeper) OnProcess(pid, eid []byte, censusRoot, censusURI string, txi
 		return
 	}
 	// If keys already exist, do nothing (this happens on the start-up block replay)
-	if len(p.EncryptionPublicKeys[k.myIndex])+len(p.CommitmentKeys[k.myIndex]) > 0 {
+	if len(p.EncryptionPublicKeys[k.myIndex]) > 0 {
```
Comment (mvdan) [inline vochain/keykeeper/keykeeper.go:197]:
```suggestion
	if len(p.EncryptionPublicKeys[k.myIndex]) > 0 { // Jordi, you might not know that you can directly suggest edits now :)
```

See the "insert suggestion" button, the left-most button on the top toolbar when you write a comment on a line.

Comment (mvdan) [reply vochain/keykeeper/keykeeper.go:197]: Meaning @jordipainan, of course

Comment (p4u) [reply vochain/keykeeper/keykeeper.go:197]: @jordipainan so let's do it via edit suggestion :)

Comment (jordipainan) [reply vochain/keykeeper/keykeeper.go:197]: hahah okii thanks!

File: vochain/keykeeper/keykeeper.go [hunk 2]
```diff
@@ -43,16 +42,14 @@ type KeyKeeper struct {
...
-	index         int8
+	pubKey  []byte
+	privKey []byte
+	index   int8
 }
 
 // Encode encodes processKeys to bytes
 func (pk *processKeys) Encode() []byte {
-	data := make([]byte, commitmentKeySize*2+encryptionKeySize*2+1)
+	data := make([]byte, +encryptionKeySize*2+1)
```
Comment (ed255) [inline vochain/keykeeper/keykeeper.go]:
```suggestion
	data := make([]byte, encryptionKeySize*2+1)
```

**Review by ed255 — APPROVED** [review]

**Pull request merged**